camelAssemblies/build.gradle

```diff
@@ -0,0 +1,47 @@
+import io.vantiq.extsrc.assy.tasks.AssemblyResourceGeneration
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation "org.apache.camel.kamelets:camel-kamelets:3.20.2"
+}
+
+ext {
+    generatedResourceBase = 'vantiqGeneratedResources'
+    zippedAssemblies = 'vantiqAssemblies'
+}
+
+tasks.withType(JavaCompile) {
+    options.encoding = 'UTF-8'
+}
+
+test {
+    useJUnitPlatform()
+}
+
```

Comment:
task to generate zipped assemblies from routes generated.  Since this is all done (based on contents of dependency), zip task doesn't work well (it's configured at configure time rather than execution time), so we use the ant zip method.